lessons/pythonClasses/lesson.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: Python Classes
+visible: true
+---
+<!-- change visible to true if you want it on the site -->
+
+# Introduction to Classes in Python
+
+ - **Authors**: Lina Tran
+ - **Research field**: Neuroscience
+ - **Lesson topic**: Object Oriented Programming with Python
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/pythonClasses>
+
+## Topics Covered:
+- Defining a Class
+- Creating Instances of a Class
+- Attributes
+ + Class Attributes vs. Normal Attributes
+- Methods
+ + Static Methods
+ + Class Methods
+- Inheritance
+
+### What You Will Need
+- Python 2.7, or 3.3+ installed
+- Download the files for [this lesson](https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/pythonClasses)
+
+If you'd like to follow along in IPython Notebook, do the following:
+ + [Jupyter Notebook](http://jupyter.readthedocs.org/en/latest/install.html)
```

Comment:
Likewise, here, the list won't show up as there is one leading space. Also, you need to have an empty line above the list to start it.